src/redux/user/user.js

```diff
@@ -32,14 +31,16 @@ export const authenticateUser = (user) => async (dispatch) => {
   })
     .then((res) => {
       if (res.status === 200) {
-        dispatch(login(res.data));
-        localStorage.setItem(
-          'authentication',
-          JSON.stringify(res.data),
-        );
+        toastify('🦄 Logged in successfully!', 'success');
+        const user = {
+          user: res.data.user,
+          token: res.data.auth_token,
+        };
+        dispatch(login(user));
+        saveState(user, 'auth');
       }
     }).catch((e) => {
-      console.log(e);
+      toastify(e.response.data.errors, 'error');
```

Comment:
The error response inside data comes like ```error``` instead of ```errors```. Thus, the message is not being displayed when I try to log in with inproper credentials